src/opscode-expander/Gemfile

```diff
@@ -2,7 +2,7 @@
 source "http://rubygems.org"
 
 gem "amqp", "~> 0.6.7"
-gem "eventmachine", '~> 0.12.10'
+gem "eventmachine", '~> 1.2.0'
```

Comment:
This is a big jump.  I think we have to do it eventually, but we'll want to keep it in mind if there are expander issues in the near future.